public/app/features/datasources/components/picker/DataSourceDropdown.tsx

```diff
@@ -359,3 +350,84 @@ function getStylesPickerContent(theme: GrafanaTheme2) {
     `,
   };
 }
+
+export interface FooterProps extends PickerContentProps {}
+
+function Footer({ onClose, onChange, onClickAddCSV, ...props }: FooterProps) {
```

Comment:
This was extracted for simplicity. Since now we're capturing the tab navigation and bringing back the focus to the input